roles/cloud_connector/README.md

```diff
@@ -0,0 +1,29 @@
+theforeman.operations.cloud_connector
+=====================================
+
+Install and configure Red Hat Cloud Connector
+
+Role Variables
+--------------
+
+Required:
+
+- `foreman_url`: The URL of the Foreman server.
+- `foreman_user`: The username cloud connector will use to talk to Foreman API.
+- `foreman_password`: The password cloud connector will use to talk to Foreman API.
```

Comment:
below (and in the playbooks) you use `foreman_cloud_connector_` prefix, I'm fine with either, but it should be consistent ;)

Reply:
Touche.